crates/bitwarden-core/src/key_management/crypto.rs

```diff
@@ -265,6 +270,63 @@ pub(super) async fn get_user_encryption_key(client: &Client) -> Result<String, C
     Ok(user_key.to_base64())
 }
 
+/// Response from the `update_kdf` function
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+#[cfg_attr(feature = "wasm", derive(Tsify), tsify(into_wasm_abi, from_wasm_abi))]
+pub struct UpdateKdfResponse {
+    /// The authentication data for the new KDF setting
+    master_password_authentication_data: MasterPasswordAuthenticationData,
+    /// The unlock data for the new KDF setting
+    master_password_unlock_data: MasterPasswordUnlockData,
+    /// The authentication data for the prior to the KDF change
```

Comment:
I feel like this comment needs to be reworded
```suggestion
    /// The authentication data for the KDF setting prior to the change
```